File: exercise01-A/src/program.py
#!/usr/bin/env python3

# Exercise 01-A: Line Totals

import sys


def main():
    for line in sys.stdin:
        print(sum([int(s) for s in line.split()]))


if __name__ == "__main__":
    main()

# vim: set sts=4 sw=4 ts=8 expandtab ft=python:

File: exercise01-A/src/program.js
#!/usr/bin/env node

// Exercise 01-A: Line Totals

const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout,
    terminal: false
});

rl.on('line', (line) => {
    console.log(line.split(" ").reduce((a, b) => Number(a) + Number(b)))
});
